plate_mapper/plate_mapper.py

```diff
@@ -127,19 +134,22 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None, special_f=None):
         # [ barcode sequence, linker primer sequence, primer plate #, well ID ]
         sample, property = '', []
```

Comment:
Too bad we didn't catch this last time.  property is also a reserved keyword.  This should be replace with something like property_